src/index.ts

```diff
@@ -54,12 +56,14 @@ export const submitCancellableForm = async (
                 if (isTerminalState(newStatus)) {
                     statusHandler(newStatus, {
                         ...formData,
```

Comment:
Medyo confusing pala itong formData na variable dito kasi may formData pa sya na attribute sa loob na naka JSON.stringify.  I rename na lang natin to form ung variable dito